src/xunit.performance.api/EtwPerformanceMetricEvaluationContext.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Parsers.Kernel;
+using Microsoft.Diagnostics.Tracing.Parsers.MicrosoftXunitBenchmark;
+using Microsoft.Xunit.Performance.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal class EtwPerformanceMetricEvaluationContext : PerformanceMetricEvaluationContext, IDisposable, IPerformanceMetricReader
```

Comment:
That sounds great!

---
In reply to: [102349184](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r102349184) [](ancestors = 102349184,102346911)